.github/filters.yaml

```diff
@@ -10,13 +10,25 @@ all-linters:
   - .trunk/trunk.yaml
   - package.json
 
+all-tools:
+  - tests/driver/**
```

Comment:
should we make this all except for lint_driver, and the reverse for `all-linters:`